Upstream: Not Applicable
Reason: Install package under name expected by other packages

--- euclid3-0.2.0.orig/setup.py
+++ euclid3-0.2.0/setup.py
@@ -8,12 +8,12 @@
 
 from distutils.core import setup
 
-setup(name='euclid',
+setup(name='euclid3',
       version=version,
       description='2D and 3D vector, matrix, quaternion and geometry module',
       author='Alex Holkner',
       author_email='alex@partiallydisassembled.net',
       url='http://partiallydisassembled.net/euclid.html',
-      py_modules=['euclid'],
+      py_modules=['euclid3'],
       )
 
--- euclid3-0.2.0.orig/test_euclid.py
+++ euclid3-0.2.0/test_euclid.py
@@ -9,7 +9,7 @@
     import pickle
 import unittest
 
-import euclid as eu
+import euclid3 as eu
 
 fe = 1.0e-5
 
